README.md

```diff
@@ -13,3 +13,47 @@ Once you've done that, check out:
 - [Q&A](https://github.com/MonacoProtocol/sdk/discussions/categories/q-a)
 
 Any troubles, issues, questions, ideas you have - anything - raise them in discussions and the community will help you out.
+
+# Introduction
+
+The Monaco Protocol is a decentralized liquidity network for exchange-based applications built on the Solana blockchain. It serves as crucial infrastructure upon which third-party front-end applications for wagering on binary outcome events can be built while accessing a shared liquidity pool.
+
+## Why is the Monaco Protocol useful?
+
+- The Monaco Protocol streamlines the wagering process.
+- It helps provide greater liquidity for participants, reduce spreads and offer lower prices.
+- It lowers the take rates of institutions and middlemen.
+- It reduces settlement times, increases transparency, and improves capital efficiency.
+
+## Why Solana?
+
+- High throughput and low costs are key pillars for any exchange-based mechanism, they are the foundation for building amazing user experiences.
+- The Solana blockchain offers theoretical TPS speeds of up to 50,000/second with costs of a fraction of a cent.
+- By building on the Solana blockchain, the Monaco Protocol delivers high speeds and low transaction costs while eliminating counter-party risk.
+
+## Is the Protocol audited?
+
+- The protocol is currently in the process of an audit.
+- Additional information on this will be released once the the audit is complete.
+
+## When Will the Protocol be on MainNet?
+
+- The protocol will launch on mainnet once auditing and testing is complete.
+- The protocol is currently deployed on the [devnet cluster](https://docs.solana.com/clusters#devnet) at [5Q2hKsxShaPxFqgVtQH3ErTkiBf8NGb99nmpaGw7FCrr](https://explorer.solana.com/address/5Q2hKsxShaPxFqgVtQH3ErTkiBf8NGb99nmpaGw7FCrr?cluster=devnet)
+- The authority of on devnet is [98CVwMftrhm6zutmV29frqRPfXsocbFnwjXVxYo7xbHX](https://explorer.solana.com/address/98CVwMftrhm6zutmV29frqRPfXsocbFnwjXVxYo7xbHX?cluster=devnet)
+
+{% hint style="danger" %} Please be vigilant when interacting with any program claiming to be the Monaco Protocol. Program addresses and authorities will always be shared by the Monaco Protocol Foundation and publicized through official channels such as these docs.{% endhint %}
+
+## How do I contribute to the Protocol?
+
+- More information about this will be available soon.
+- For now you can share ideas over on the [DevHub](https://github.com/MonacoProtocol/sdk/discussions).
+
+## Have any applications built on the Monaco Protocol?
+
+- BetDEX exchange is currently in closed beta.
+- You can learn more about the exchange on https://betdex.gitbook.io/betdex-beta/.
+
```

Comment:
Can we add that community members from SAMO are a crypto-based prediction market?

Reply:
adding!